### ISSUE TITLE
Recent release of svelte breaks this plugin

### ISSUE THREAD

**Issue by maraisr** [sveltejs/rollup-plugin-svelte#152]
There seems to be a "autoload" method doing:

```js
function autoload() {
	const pkg = require('svelte/package.json');
	const version = to_major(pkg.version);

	const { compile, preprocess } = require(version >= 3 ? 'svelte/compiler.js' : 'svelte');
	return { compile, preprocess, version };
}
```

which seems to complain like:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './compiler.js' is not defined by "exports" in /node_modules/svelte/package.json
```

node: 15.0.1
yarn: 1.22.5
svelte: 3.29.7
rollup-plugin-svelte: 1.4.1
svelte-preprocess: 4.6.0

**Comment by Conduitry** [comment]
There's no rollup-plugin-svelte@1.4.1, so I'm not sure what version you're really using - but this was already fixed in rollup-plugin-svelte 6.1.1 via #151.

**Comment by maraisr** [comment]
My apologizes @Conduitry it was meant to read: 6.1.0. Have updated.
